### ISSUE TITLE
Directly Fetching Translations from Database Without Using .ftl Files

### ISSUE THREAD

**Issue by NCura** [mondeja/leptos-fluent#256]
Hello!

I'm working on a project where I need to build a back-office system so my client can independently update certain website content, including text changes.

I’m considering fetching translations from the database, writing them to an `.ftl` file, and then retrieving them for use. However, I’m wondering if there’s a straightforward way to fetch translations directly from the database without relying on an `.ftl` file. 

Any guidance or suggestions on this would be greatly appreciated! 

Thank you!

**Comment by mondeja** [comment]
Nope, there is currently nothing similar implemented. Do you require updates at runtime?

**Comment by NCura** [comment]
Yes, the goal is to deploy the server and allow the client to authenticate within the website's admin section. From there, they can submit forms to update text directly in the database, with these changes reflected in real-time on the public-facing website.

**Comment by mondeja** [comment]
PRs welcome. Related to #110.

**Comment by mondeja** [comment]
Closing this as a duplicate in favour of #110...
